blog/docs/articles/images/iter8-grpc.png

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
this doesn't like a png?

Reply:
I had placed the images in the wrong subfolder. Hopefully, fixed now.

Reply:
`<img src="../images/iter8-grpc.png" alt="gRPC performance test" width="800"/>`